librispect/features/predict.py

```diff
@@ -50,4 +50,11 @@ def batch_iter(self, path_list, batch_size):
                     yield [term_batch[idxs, ...], pterm_batch[idxs, ...]], labels[
                         idxs, ...
                     ]
+
+    def split_validation(self, path_list, validation_percentage):
```

Comment:
isn't this already in utils.py?

Reply:
yeah, should the script import from there instead?